src/MoveUtil.hs

```diff
@@ -0,0 +1,35 @@
+module MoveUtil where
```

Comment:
Probably, something wrong happened on communication level. But this `MoveUtil` module shouldn't exist. Basically, all tasks from issue #26 won't touch anything related to `fix-point` at all. So you don't need to copy functionality of `fix-point` or move `fix-point` module somewhere else.